docs/azure/ContinuousDeployment.rst

```diff
@@ -24,60 +24,80 @@ To enable continuous deployment, complete the following steps:
+     * ``AZURE_SUBSCRIPTION_ID`` - Store the ID of the subscription which contains the nRF Asset Tracker resources.
 
-      az account set --subscription $SUBSCRIPTION_ID
-      # Verify that it is set to default
-      az account list --output table
+     Set the following following values from your :file:`.envrc` file as secrets as well:
 
-#. Create the CI credentials:
+     * ``RESOURCE_GROUP``
+     * ``LOCATION``
+     * ``APP_NAME``
+     * ``B2C_TENANT``
+     * ``APP_REG_CLIENT_ID``
 
-   .. code-block:: bash
+     If you have enabled the :ref:`azure-unwired-labs-cell-geolocation`, add your API key as a secret as well:
 
-      az ad sp create-for-rbac --name 'https://nrfassettracker.invalid/cd' --role contributor \
-         --scopes \
-            "/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP:-nrfassettracker}" \
-         --sdk-auth \
-         > cd-credentials.json
+     * ``UNWIRED_LABS_API_KEY``
 
-Provide the credentials to GitHub Actions
-*****************************************
+   - Set the secrets using the GitHub CLI:
 
-1. Add the following `secrets <https://docs.github.com/en/rest/reference/actions#secrets>`_ to an `environment <https://docs.github.com/en/actions/reference/environments#creating-an-environment>`_ called ``production`` in your fork of the nRF Asset Tracker for Azure:
+     Alternatively, you can use the `GitHub CLI <https://cli.github.com/>`_  with the environment settings from above (make sure to create the ``production`` `environment <https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment>`_ in your repository first):
 
-   * ``AZURE_CREDENTIALS`` - Store the contents of the JSON file created in the above step.
-  
-#. Add the following following values from your :file:`.envrc` file as secrets as well:
+    .. code-block:: bash
 
-   * ``RESOURCE_GROUP``
-   * ``LOCATION``
-   * ``APP_NAME``
-   * ``B2C_TENANT``
-   * ``APP_REG_CLIENT_ID``
+       export AZURE_CLIENT_ID=`az ad app list | jq -r '.[] | select(.displayName=="https://nrfassettracker.invalid/cd") | .appId' | tr -d '\n'`
+       export AZURE_TENANT_ID=`az ad sp show --id ${AZURE_CLIENT_ID} | jq -r '.appOwnerOrganizationId' | tr -d '\n'`
+       gh secret set AZURE_CLIENT_ID --env production --body "${AZURE_CLIENT_ID}"
+       gh secret set AZURE_TENANT_ID --env production --body "${AZURE_TENANT_ID}"
+       gh secret set AZURE_SUBSCRIPTION_ID --env production --body "${SUBSCRIPTION_ID}"
+       gh secret set RESOURCE_GROUP --env production --body "${RESOURCE_GROUP}"
+       gh secret set LOCATION --env production --body "${LOCATION}"
+       gh secret set APP_NAME --env production --body "${APP_NAME}"
+       gh secret set B2C_TENANT --env production --body "${B2C_TENANT}"
+       gh secret set APP_REG_CLIENT_ID --env production --body "${APP_REG_CLIENT_ID}"
+
+#. Grant the application created in step 1 Owner permissions for your resource group:
```

Comment:
Could we add a label before step 1 and link to to it for Step 1? Something like 
.. _step1_authentication_openid: before line 32 
and referencing step 1 as :ref:`step 1 <step1_authentication_openid>`